pom.xml

```diff
@@ -164,9 +164,11 @@
           <target>1.7</target>
           <excludes>
             <exclude>com/datalogics/pdf/samples/extraction/TextCharacterBoxes.java</exclude>
+	    <exclude>com/datalogics/pdf/samples/forms/ExportFormDataToCsv.java</exclude>
```

Comment:
This is the part of the Jenkins log that tells you that the POM was changed during building. It's near the end:

```
[workspace] $ /bin/sh -xe /tmp/hudson4715329650434021097.sh
+ git diff --stat --exit-code
 pom.xml | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)
+ echo '[ERROR] Build changed some Git files'
[ERROR] Build changed some Git files
+ exit 1
Build step 'Execute shell' marked build as failure
```